src/System.Private.DataContractSerialization/src/System/Xml/PrefixHandle.cs

```diff
@@ -178,8 +179,10 @@ public int CompareTo(PrefixHandle that)
             return GetString().CompareTo(that.GetString());
         }
 
-        private bool Equals2(PrefixHandle prefix2)
+        public bool Equals(PrefixHandle prefix2)
         {
+            if (ReferenceEquals(prefix2, null))
+                return false;
```

Comment:
Also in this class `==` is overloaded so it would have to at least be `if ((object)prefix2 == null)`. The chain of equality methods here is a bit convoluted IMO, but I wanted to make each change as a single simple easy-win change rather than move anything around beyond that.